### ISSUE TITLE
How to test the model on the MPII test-set?not the valid-set.

### ISSUE THREAD

**Issue by PCRKTY** [leoxiaobin/deep-high-resolution-net.pytorch#216]
I want to test model on the MPII est-set, but the config file /experiments/mpii/hrnet/w32_256x256_adam_lr1e-3.yaml is only for testing on validation set.
And I tried to covert /output/pred.mat to "pred_keypoints_mpii.mat" to run the evaluatePCKh.m of MPII Evaluation toolkit, but it failed. 
So, I have questions: 
How to test the model on the MPII test-set? 
And how to get the results of PCKh?
Thank you for your great work!

**Comment by newwhitecheng** [comment]
@PCRKTY @leoxiaobin  so how do you test on test set?

**Comment by Aliiiu** [comment]
Pls once again, share how you were able to solve this
